rinja_derive/src/generator.rs

```diff
@@ -997,6 +997,7 @@ impl<'a> Generator<'a> {
                     _ => Ok(false),
                 }
             }
+            Target::Placeholder("_" | "..") => Ok(false),
```

Comment:
The second case will never be true, though. `Target::Placeholder(_)` only ever captures `"_"`, and `Target::Rest(_)` captures `..` or `.. @ var`.

Both variant should probably contain a `WithSpan<'a, ()>` instead of the captured token, to make the meaning more clear.

Reply:
Ok, gonna clean up this code because there is no point in keeping content if it's not used.